daily_read/order_portal.py

```diff
@@ -17,26 +18,27 @@ class OrderPortal(object):
     """Class to handle NGI order portal interaction"""
 
     def __init__(self):
-        base_url = config_values.DAILY_READ_ORDER_PORTAL_URL
-        api_key = config_values.DAILY_READ_ORDER_PORTAL_API_KEY
+        base_url = config_values.ORDER_PORTAL_URL
+        api_key = config_values.ORDER_PORTAL_API_KEY
 
         if base_url is None:
-            raise ValueError("environment variable DAILY_READ_ORDER_PORTAL_URL not set")
+            raise ValueError("environment variable ORDER_PORTAL_URL not set")
```

Comment:
The env variable is still called `DAILY_READ_ORDER_PORTAL_URL` and its the config variable that's called `ORDER_PORTAL_URL`?